EasyPost/EasyPost.csproj

```diff
@@ -10,7 +10,7 @@
     <Configurations>Release;Debug;Linting</Configurations>
     <Platforms>AnyCPU</Platforms>
     <PackageId>EasyPost</PackageId>
-    <LangVersion>9</LangVersion>
+    <LangVersion>10</LangVersion>
```

Comment:
issue: we shouldn't be bumping dependencies along with code changes, these are mixing too many concerns and should typically happen separately

Reply:
Not a dependency